services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+
+    order = Order.objects.create(
+        user=get_user_model().objects.get(username=username)
+    )
+    if date:
+        order.created_at = date
+    order.save()
+
+    for ticket_data in tickets:
+        Ticket.objects.create(
+            row=ticket_data["row"],
+            seat=ticket_data["seat"],
+            movie_session_id=ticket_data["movie_session"],
+            order=order,
+        )
+
+
+def get_orders(username: str = None) -> QuerySet:
+    if username:
+        return Order.objects.filter(user__username=username)
+    return Order.objects.all()

```

Comment:
make a variable where you get all `Orders` and then, if `username is not None` apply filters. With such approach you can get rid of one `return` statement